src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,21 @@
+export const TodoForm = () => {
+  return (
+    <header className="todoapp__header">
+      {/* this buttons is active only if there are some active todos */}
```

Comment:
remove comments, it's redundant in production
nice code - code that don't need to be commented